indico/modules/events/registration/templates/management/_registration_details.html

```diff
@@ -223,7 +223,8 @@
                 data-update="#registration-details"
                 data-method="POST"
                 data-href="{{ url_for('.reset_registration', registration) }}"
-                data-confirm="{% trans %}Are you sure you want to reset this registration status?{% endtrans %} {{ confirm_notification }}"
+                data-confirm="{% trans %}Are you sure you want to reset this registration status?{% endtrans %}
+                              {% block reset_registration_button_notification_text %}{{ confirm_notification }}{% endblock %}"
```

Comment:
```suggestion
                              {%- block reset_registration_button_notification_text %} {{ confirm_notification }}{% endblock %}"
```

Might be nicer to avoid the linebreak in the final html...

Reply:
or try if you can put the block inside the `{% set ... %}` above. that'd be more readable IMHO